src/EnergyPlus/WaterUse.cc

```diff
@@ -1118,47 +1119,63 @@ namespace WaterUse {
-                    ShowContinueErrorTimeStamp(state, "");
-                    ShowContinueError(state, format("...hot water temperature       = {:.3R} C", this->HotTemp));
-                    ShowContinueError(state, format("...cold water temperature       = {:.3R} C", this->ColdTemp));
-                    ShowContinueError(state, "...Note: hot water temperature should be greater than or equal to the cold water temperature");
-                    ShowContinueError(state,
-                                      "...Hot water temperature should be greater than or equal to the cold water temperature. "
-                                      "Verify temperature setpoints and schedules.");
-                } else {
-                    ShowRecurringWarningErrorAtEnd(
-                        state,
-                        this->Name + "\" - Hot water temperature should be greater than or equal to the cold water temperature error continues...",
-                        this->HWTempErrIndex,
-                        this->HotTemp,
-                        this->HotTemp);
+                if (!state.dataGlobal->WarmupFlag) {
+                    // print error for variables of hot water temperature
+                    ++this->CWHWTempErrorCount;
+                    if (this->CWHWTempErrorCount < 2) {
+                        ShowWarningError(
+                            state,
+                            format("CalcEquipmentFlowRates: \"{}\" - Hot water temperature is less than the cold water temperature ({:.2R} C)",
+                                   this->Name,
+                                   TempDiff));
+                        ShowContinueErrorTimeStamp(state, "");
+                        ShowContinueError(state, format("...hot water temperature       = {:.2R} C", this->HotTemp));
+                        ShowContinueError(state, format("...cold water temperature       = {:.2R} C", this->ColdTemp));
+                        ShowContinueError(state, "...Note: hot water temperature should be greater than or equal to the cold water temperature");
+                        ShowContinueError(state,
+                                          "...Hot water temperature should be greater than or equal to the cold water temperature. "
+                                          "Verify temperature setpoints and schedules.");
+                    } else {
+                        ShowRecurringWarningErrorAtEnd(
+                            state,
+                            format("\"{}\" - Hot water temperature should be greater than or equal to the cold water temperature error continues...",
+                                   this->Name),
+                            this->CWHWTempErrIndex,
+                            TempDiff,
+                            TempDiff);
+                    }
                 }
             } else if ((this->TargetTemp - this->HotTemp) > EPSILON) {
```

Comment:
I wonder if these should be separate if statements. It strikes me that they aren't mutually exclusive. i.e., you could have a target temperature that is colder than the hot water temperature that is colder than the cold water temperature. The question is if we want to show all three warnings in this scenario.